Source/ConversationV1/Models/DialogNodeNextStep.swift

```diff
@@ -21,7 +21,6 @@ public struct DialogNodeNextStep {
 
     /// How the `next_step` reference is processed.
     public enum Behavior: String {
-        // swiftlint:disable:next identifier_name
```

Comment:
We aren't generating this line for the linter yet. So it makes sense that it wasn't in your generated PR. I had to manually keep this change when committing to git.